src/payment/data/reducers.js

```diff
@@ -1,11 +1,14 @@
+import { combineReducers } from 'redux';
+
 import { FETCH_BASKET } from './actions';
+import { reducer as coupon } from '../coupon';
 
-export const initialState = {
-  loading: true,
+export const basketInitialState = {
+  loading: false,
   loadingError: null,
 };
 
-const reducer = (state = initialState, action) => {
+const basket = (state = basketInitialState, action = null) => {
```

Comment:
This `basket` reducer is a catch-all for all the data that comes back that hasn't yet been split out into its own sub-modules, such as `summary` or `products`, perhaps.  Point being, I wanted to move it out of the way of `coupon` start using combineReducers below, but haven't yet messed with what it actually does.